src/components/profile/Profile.js

```diff
@@ -104,14 +105,26 @@ const ProfileContainer = styled.div`
   justify-content: center;
   margin-top: -4.5%;
   margin-bottom: 10%;
+  @media ${device.laptop} {
+    margin-top: -4%;
```

Comment:
margin-top should be -7% here to make sure the banner image is lined up against the navbar